scripts/publish-instance-metrics/amazon-cloudwatch-agent.json

```diff
@@ -0,0 +1,37 @@
+{
+  "metrics": {
+    "namespace": "SageMakerNotebookInstances",
+    "metrics_collected": {
```

Comment:
This will put metrics for all Notebook Instances into one namespace. How about adding a dimension as the Notebook Instance Name ?